src/blob/index/core.rs

```diff
@@ -99,12 +108,12 @@ where
             ..
         } = &mem;
         let len = self.headers.len();
-        trace!("record_header_size: {}, records_allocated: {}, data.len(): {}, entry_size (key + vec): {}",
-                MemoryAttrs::<K>::RECORD_HEADER_SIZE, records_allocated, len, MemoryAttrs::<K>::BPTREE_ENTRY_SIZE);
+        trace!("record_header_size: {}, records_allocated: {}, data.len(): {}",
```

Comment:
I think, there is no need to log the constant values. Perhaps, it will be better to log `len`, `records_allocated` and calculated memory after the calculation happened